R/create_tables_doc.R

```diff
@@ -27,7 +27,8 @@ create_tables_doc <- function(resdir = NULL,
     tables_doc <- paste0(
       tables_doc,
       add_chunk(
-        paste0("rda_dir <- '", rda_dir, "/rda_files'"),
+        paste0("library(flextable)
```

Comment:
This and the following line will be on the same line when cat
```suggestion
        paste0("library(flextable)", "\n", 
```